pinot-common/src/main/java/org/apache/pinot/common/datablock/DataBlockUtils.java

```diff
@@ -102,19 +97,6 @@ private DataBlockUtils() {
 
   static final int VERSION_TYPE_SHIFT = 5;
 
-  public static MetadataBlock getErrorDataBlock(Exception e) {
-    if (e instanceof ProcessingException) {
-      int errorCodeId = ((ProcessingException) e).getErrorCode();
-      return getErrorDataBlock(Collections.singletonMap(errorCodeId, extractErrorMsg(e)));
-    } else if (e instanceof QueryException) {
-      int errorCodeId = ((QueryException) e).getErrorCode().getId();
-      return getErrorDataBlock(Collections.singletonMap(errorCodeId, extractErrorMsg(e)));
-    } else {
-      // TODO: Pass in meaningful error code.
-      return getErrorDataBlock(Map.of(QueryErrorCode.UNKNOWN.getId(), extractErrorMsg(e)));
-    }
-  }
```

Comment:
This and `getErrorDataBlock(Map<Integer, String> exceptions)` are not used anymore